daemons/controld/controld_messages.c

```diff
@@ -853,8 +856,8 @@ handle_node_info_request(xmlNode *msg)
         crm_xml_add(reply, XML_ATTR_UUID, node->uuid);
         crm_xml_add(reply, XML_ATTR_UNAME, node->uname);
         crm_xml_add(reply, XML_NODE_IS_PEER, node->state);
-        crm_xml_add_boolean(reply, XML_NODE_IS_REMOTE,
-                            node->flags & crm_remote_node);
+        pcmk__xe_set_bool_attr(reply, XML_NODE_IS_REMOTE,
+                               node->flags & crm_remote_node);
```

Comment:
may as well use pcmk_is_set() here

Reply:
Fixed.  Note that right above here is a block of crm_xml_add that could be replaced with a call to pcmk__xe_set_props in some other PR.  Also, this could be a spot to add your function that sets multiple types of attrs in a single go.